src/diffusers/models/unet_2d_condition.py

```diff
@@ -262,6 +272,7 @@ def forward(
         timestep: Union[torch.Tensor, float, int],
         encoder_hidden_states: torch.Tensor,
         class_labels: Optional[torch.Tensor] = None,
+        additional_embeddings: Optional[torch.Tensor] = None,
```

Comment:
What are additional_embeddings? Ok for now but should get a better name down the road IMO

Reply:
Yep agreed a different name would be better. Something like `additive_time_embeddings` maybe?

Context on this is clip projects the text and image embeddings to the time embedding dimension and then adds them to the time embeddings

See this annotated snippet

https://github.com/huggingface/diffusers/pull/1428/files#diff-a3969850f2ad577ffdefb08c6aac90db4b3ddf08cebfafe3f7e23b6e169ad997R56-R66

Reply:
`additive_time_embeddings` sounds great!

Reply:
done!